### ISSUE TITLE
[SCREENREADER]: Decorative content MUST be implemented in a way it can be ignored by assistive technology

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#8257]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

On the cards underneath Top Tasks there are blue lines beneath the headings. These lines **should** be suppressed from being read out by assistive technology. Currently, each of these blue lines is read as "horizontal separator" by the screen reader.

If non-text content is pure decoration, is used only for visual formatting, or is not presented to users, then it **must** be implemented in a way that it can be ignored by assistive technology.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to read the content of the page without extraneous noise about decorative visual elements.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: all
* Server destination: production

## Steps to Recreate

1. Enter https://www.va.gov/find-forms in browser
2. Start screenreading device listed in Environment
3. Navigate to the Top Tasks
4. Verify that the horizontal line is read by the screen reader

## Possible Fixes (optional)
Add ` aria-hidden="true" role="presentation"` to the `<hr>` elements.

### Current code
```html
<hr class="featured-content-hr vads-u-margin-y--1p5 vads-u-border-color--primary">
```

### Recommendation
```diff
+ <hr aria-hidden="true" role="presentation" class="featured-content-hr vads-u-margin-y--1p5 vads-u-border-color--primary">
```

## WCAG or Vendor Guidance (optional)

* [W3C WCAG 1.1.1 - Failure of Success Criterion 1.1.1 due to not marking up decorative images in HTML in a way that allows assistive technology to ignore them](https://www.w3.org/TR/WCAG20-TECHS/F38.html)

* [W3C Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)

* [HTML5 Accessibility: aria-hidden and role=”presentation”](http://john.foliot.ca/aria-hidden/)

## Screenshots or Trace Logs

![Screen Shot 2020-04-21 at 8 46 20 PM](https://user-images.githubusercontent.com/57469/79928867-2435ba00-8412-11ea-863e-748556306462.png)

**Comment by 1Copenut** [comment]
@jenstrickland I'd call this a `3` or `4`. It's information a screen reader user doesn't need, but it's not disrupting the flow of information.

**Comment by kelsonic** [comment]
Whoops this is a find-forms a11y ticket, did you mean to make this when making the YR a11y tickets @jenstrickland?

**Comment by kelsonic** [comment]
Sorry, ignore my comment lol

**Comment by stephorkandatx** [comment]
@jenstrickland this will be ready for you to validate as you have time. Thank you!

**Comment by jenstrickland** [comment]
@stephorkandatx In staging I see:
`<hr class="featured-content-hr vads-u-margin-y--1p5 vads-u-border-color--primary">`, and expect to see `<hr aria-hidden="true" role="presentation" class="featured-content-hr vads-u-margin-y--1p5 vads-u-border-color--primary">`.  Is there a PR waiting to be merged?

**Comment by kelsonic** [comment]
I don't think it's been deployed yet @jenstrickland @stephorkandatx 🙂Just merged recently

**Comment by stephorkandatx** [comment]
@kelsonic can you confirm the PR was merged?

**Comment by kelsonic** [comment]
Merged! Validated on production: https://va.gov/find-forms

![image](https://user-images.githubusercontent.com/12773166/82344243-09267d80-99b1-11ea-8b4c-01729ecd8106.png)

**Comment by jenstrickland** [comment]
Validated. Thanks for the great work, y'all! 😄
